deploy/cron.js

```diff
@@ -1,3 +1,8 @@
+if (process.env.NODE_ENV === "production") {
```

Comment:
This tool is used by others, who may not be using New Relic. I don't think we should tie the use of New Relic solely to whether the app is running in a production environment. Using a `NODE_ENV` production flag is also a good idea generally for things we don't need to run in development (though I think this is the only such thing right now).

So, I recommend that this hinge on a `NEW_RELIC_ENABLED` (or similar) env var instead, to make it more semantically obvious what the deployer is opting in to.